### ISSUE TITLE
Can't change headerBackTitle

### ISSUE THREAD

**Issue by buraks** [expo/router#539]
### Which package manager are you using? (Yarn is recommended)

yarn

### Summary

I have a very simple app with one home page and one sub-page.

I want to hide the "Back" text on the header on iOS. Normally on react-navigation, there are `headerBackTitle` and `headerBackTitleVisible`, which I could use to change/hide this.

On expo-router I am unable to change it or hide it. Could you please kindly check?

### Minimal reproducible example

This is the file structure I have:

```
app/
├─ _layout.tsx
├─ index.tsx
├─ chat/
│  ├─ [id].tsx
```

Here are the contents:

`_layout.tsx`
```tsx
import { Stack } from 'expo-router';

export default function Layout() {
  return <Stack />;
}
```

`index.tsx`
```tsx
import { Link } from 'expo-router';

export default function Home() {
  <SafeAreaView style={styles.container}>
    (...)
    <Link key={id} href={`/chat/${id}`} asChild>
      <Pressable>
        (...)
      </Pressable>
    </Link>
  </SafeAreaView>
}
```

`chat/[id].tsx`
```tsx
import { Stack, useLocalSearchParams } from 'expo-router';

export default function ChatView() {
  const { id } = useLocalSearchParams<{ id: string }>();
  (...)
  return (
    <View style={styles.container}>
      <Stack.Screen
        options={{
          title: `Test`,
          headerBackTitle: 'Something',
          headerBackTitleVisible: false,
        }}
      />
      (...)
    </View>
  )
}
```

## The Result

<img src="https://user-images.githubusercontent.com/2693237/236329785-65039a7e-020c-4389-b793-2c5b74f51cfb.jpg" width="500" />

Basically, the title can be changed. However, the back title can't be modified.

I am using:

```
"expo": "^48.0.7",
"expo-router": "^1.5.3",
"react-native": "0.71.7",
"react-native-safe-area-context": "4.5.0",
"react-native-screens": "~3.20.0",
```

**Comment by w00ing** [comment]
Experiencing the same issue: 


```
"expo": "~48.0.11",
"expo-router": "^1.5.3",
"react-native": "0.71.7",
"react-native-safe-area-context": "4.5.0",
"react-native-screens": "~3.20.0",
```

**Comment by geshub** [comment]
Also looking for a way to update the default 'Back' text on IOS

**Comment by JoshBot-Debug** [comment]
Try this 

`_layout.tsx`

```ts
import { Stack } from 'expo-router';

export default function Layout() {
  return (
   <Stack>
      <Stack.Screen
        name="(chat)/[id]"
        options={{
          title: `Test`,
          headerBackTitle: 'Something',
          headerBackTitleVisible: false,
        }}
      />
  </Stack>
  );
}
```

`chat/[id].tsx`
```ts
import { Stack, useLocalSearchParams } from 'expo-router';

export default function ChatView() {
  const { id } = useLocalSearchParams<{ id: string }>();
  (...)
  return (
    <View style={styles.container}>
      (...)
    </View>
  )
}
```

**Comment by steve228uk** [comment]
This was an issue with `react-native-screens` and not `expo-router` https://github.com/software-mansion/react-native-screens/pull/1726

**Comment by EvanBacon** [comment]
Closing as we cannot fix this in expo-router without upstream changes.
